webapp/src/ts/app.component.ts

```diff
@@ -633,14 +633,38 @@ export class AppComponent implements OnInit {
       'boot_time:1:to_first_code_execution',
       window.startupTimes.firstCodeExecution - window.startupTimes.start
     );
+
+    if (window.startupTimes.replicationStarted && window.startupTimes.replicationEnded) {
+      this.telemetryService.record(
+        'boot_time:2_1:to_replication',
+        window.startupTimes.replicationEnded - window.startupTimes.replicationStarted
+      );
+    }
+
+    if (window.startupTimes.purgeStarted && window.startupTimes.purgeEnded) {
```

Comment:
I'm trying to think of other records for telemetry, @dianabarsan do you think is useful to log when [translations are loaded](https://github.com/medic/cht-core/blob/d6c83f92168a80b5ebdb7409f8ac61909a54928c/webapp/src/ts/app.component.ts#L147) or when the [db has finished setting up](https://github.com/medic/cht-core/blob/d6c83f92168a80b5ebdb7409f8ac61909a54928c/webapp/src/ts/app.component.ts#L181).

Reply:
`setupDb` it looks like we're just subscribing to events in that method. Am I missing something? 
On loading translations, I _think_ we should definitely add which language the user is using to telemetry! I think there's a separate issue about that: https://github.com/medic/cht-core/issues/7089  Loading time is less important maybe? it should be quite fast. How long does it take?

Reply:
yeah, there isn't much in `setupDb` maybe the must relevant thing is the update of replication status but not sure if we need to record that for analysis.

The translations is very fast and I like the idea of logging the language, I'm going to record a telemetry in `SetLanguageService.set` function

Reply:
There already is a bunch of telemetry around replication: https://docs.communityhealthtoolkit.org/apps/guides/performance/telemetry/

Reply:
Done